projects/js-toolkit/packages/liferay-cli/bin/liferay.js

```diff
@@ -6,11 +6,18 @@
  */
 
 const {argv} = require('yargs')
-	.option('version', {
```

Comment:
only leaving a comment because the description of this commit a thing of beauty 😍

Reply:
I'm curious, when you create long commit descriptions [like this](https://github.com/liferay/liferay-frontend-projects/pull/584/commits/03feff7646859b9cf447029d441b19e80e092f0f), do you just write it all from your CLI or write it somewhere else first?

Reply:
I use `git gui` to make the commits. It comes with git, it's visual enough for me, and works in every platform :sweat_smile: . It has an edit field that is more or less 80 chars wide, and that helps you when it comes to cutting lines.

You can also use `vim` to edit messages, and it will wrap at 80 columns too and will let you edit the message in a powerful way.